### PR TITLE
Added: Mate Common

### PR DESCRIPTION
**Pull request by Toasterson**
Initial minimal working Version of Mate 1.14

### CODE REVIEW

File: components/desktop/mate/mate-common/Makefile [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+build:		$(BUILD_32_and_64)
+
+install:	$(INSTALL_32_and_64)
+
+test:		$(NO_TESTS)
+
+BUILD_PKG_DEPENDENCIES = $(BUILD_TOOLS)
+REQUIRED_PACKAGES += developer/documentation-tool/gtk-doc
+
+include $(WS_TOP)/make-rules/depend.mk
```
Comment (pyhalov) [inline components/desktop/mate/mate-common/Makefile]: remove BUILD_PKG_DEPENDENCIES and depend.mk

File: components/desktop/mate/mate-common/Makefile [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+CONFIGURE_OPTIONS+=	--disable-scrollkeeper
+CONFIGURE_OPTIONS+=	--localstatedir=/var/lib
+
+CONFIGURE_ENV+= PYTHON="$(PYTHON)"
+
+COMPONENT_BUILD_ENV += CC="$(CC)"
+COMPONENT_BUILD_ENV += CFLAGS="$(CFLAGS)"
+
+# common targets
+build:		$(BUILD_32_and_64)
```
Comment (pyhalov) [inline components/desktop/mate/mate-common/Makefile]: why build for both architectures if component seems to consist only from macroses and scripts?

File: components/desktop/mate/mate-common/Makefile [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+include $(WS_TOP)/make-rules/configure.mk
+include $(WS_TOP)/make-rules/ips.mk
+
+PATH=/usr/gnu/bin:/usr/bin
+
+COMPONENT_PREP_ACTION = cd $(@D) && NOCONFIGURE=1 ./autogen.sh
+
+
+CONFIGURE_OPTIONS+=	--sysconfdir=/etc
+CONFIGURE_OPTIONS+=     --libexecdir=/usr/lib/mate
```
Comment (pyhalov) [inline components/desktop/mate/mate-common/Makefile]: please, fix formatting

File: components/desktop/mate/mate-common/Makefile [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+CONFIGURE_OPTIONS+=	--disable-static
+CONFIGURE_OPTIONS+=	--disable-scrollkeeper
+CONFIGURE_OPTIONS+=	--localstatedir=/var/lib
```
Comment (pyhalov) [inline components/desktop/mate/mate-common/Makefile]: WARNING: unrecognized options: --disable-static, --disable-scrollkeeper

File: components/desktop/mate/mate-common/Makefile (hunk 2 above)
Comment (Toasterson) [inline components/desktop/mate/mate-common/Makefile]: None really. i just try to build all components for both architectures. 
Should i change it to build 64 bit only?

Comment (pyhalov) [inline components/desktop/mate/mate-common/Makefile]: We still default to $(BUILD_32) if there are no strong reasons not to. But in this case it doesn't really matter.

File: components/desktop/mate/mate-common/Makefile [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+include $(WS_TOP)/make-rules/configure.mk
+include $(WS_TOP)/make-rules/ips.mk
+
+PATH=/usr/gnu/bin:/usr/bin
+
+COMPONENT_PREP_ACTION = cd $(@D) && NOCONFIGURE=1 ./autogen.sh
+
+
+CONFIGURE_OPTIONS+=	--sysconfdir=/etc
+CONFIGURE_OPTIONS+= --libexecdir=/usr/lib/mate
```
Comment (pyhalov) [inline components/desktop/mate/mate-common/Makefile:42]: there are still some issues with formatting

File: components/desktop/mate/mate-common/Makefile [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+# common targets
+build:		$(BUILD_32)
+
+install:	$(INSTALL_32)
+
+test:		$(NO_TESTS)
+
+REQUIRED_PACKAGES += developer/documentation-tool/gtk-doc
+
```
Comment (pyhalov) [inline components/desktop/mate/mate-common/Makefile]: Please, remove this empty line and merge commits

**Pull request merged**